src/content/docs/fr/2/guide/frontend/nextjs.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: Next.js
+i18nReady: true
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import CommandTabs from '@components/CommandTabs.astro';
+
+Next.js est un framework de React. Apprenez-en plus au sujet de Next.js sur https://nextjs.org. Ce guide est valable à partir de Next.js 13.4.19.
+
+## Liste des Vérifications

```

Comment:
```suggestion
## Liste de contrôle
```

Reply:
I could not find an exact translation, thank you ! However doesn't you think that we need to keep the capital letter at the begining of the words like in all titles ?